.github/workflows/test.yml

```diff
@@ -101,6 +104,75 @@ jobs:
 
           skopeo copy oci-archive:dotnet-aspnet.tar docker-daemon:${{ env.aspnet-image-name }}
 
+      - name: Generate SBOM for the .NET runtime deps container image
+        run: |
+          set -x
+          archs=`cat rockcraft.*.dotnet-deps-7.0-${{ matrix.ubuntu-release }}.yaml | shyaml get-value platforms | shyaml keys`
```

Comment:
please use a step output, otherwise we'll have this parsing duplicated everywhere and that's a maintenance nightmare